src/webserver.mm

```diff
@@ -14,21 +14,37 @@
 
 namespace {
     std::thread g_server;
-
+    fostlib::http::server *g_webserver = nullptr;
     file_loader_callback file_loader;
 }
 
 
 extern "C" void webserver_start() {
     /// Start the web server and set the termination condition
     g_server = std::thread{[]() {
-        fostlib::http::server server(fostlib::host(0), 2555);
-        server(fostlib::urlhandler::service, []() -> bool {
-            return false;
-        });
+        try {
```

Comment:
Might be best to insert a call to `webserver_stop` at this point so even if the API is used wrong it will still do the right thing.

Reply:
Do you mean insert webserver_stop before return false?

Reply:
I add webserver_stop before starting thread g_sever, is this correct?